cli/src/actions/loadPaths.js

```diff
@@ -22,6 +22,7 @@ export default async (catalogSrcDir: string, catalogBuildDir: string, framework:
   appSrc: resolveAppPath('src'),
   yarnLockFile: resolveAppPath('yarn.lock'),
   babelrc: resolveAppPath('.babelrc'),
+  postCSSConfig: resolveAppPath('.postcssrc') || resolveAppPath('postcss.config.js') || resolveAppPath('.postcssrc.js'),
```

Comment:
This doesn't work 😅 . `postCSSConfig` will _always_ be the first value, even if none of those files exists (resolving a path doesn't ensure that it points to something). Even if it _did_ work, the behavior will be different from postcss-loader's (e.g. it won't pick up `.postcssrc.yaml` etc).